template_tests/filters.tpl.out

```diff
@@ -22,13 +22,13 @@ cut
 Helloworld
 
 default
-n/a
+<map[string]interface {} Value>
```

Comment:
@nishantsharma  @sprksh not sure at what point we introduced these changes, but this is the current output of the tests